components/Old_Join.js

```diff
@@ -1,21 +1,17 @@
-import { FaDiscord } from "react-icons/fa";
+import { FaDiscord, FaPlayCircle } from 'react-icons/fa'
 const Join = () => {
   return (
-    <main className="py-6 bg-white">
-      <div className="max-w-7xl mx-auto px-4 sm:px-6 lg:px-8">
-        <div className="lg:text-center">
-          <div className="inline-flex rounded-md shadow">
-            <a
-              href="https://discord.com/invite/TcmA2kbJeA"
-              className="inline-flex items-center justtify-center px-5 py-3 border border-transparent
-                text-base font-medium rounded-md text-white bg-red-600 hover:bg-red-800"
-            >
-              <span className="px-1">
-                <FaDiscord />
-              </span>
-              Join the Community!
-            </a>
-          </div>
+    <main className='py-6 bg-white'>
+      <div className='max-w-7xl mx-auto px-4 sm:px-6 lg:px-8'>
+        <div className='flex gap-4'>
+          <a href='https://discord.com/invite/TcmA2kbJeA' className='btn btn-primary'>
+            <FaDiscord />
+            Join the Community!
+          </a>
+          <a href='https://discord.com/invite/TcmA2kbJeA' className='btn'>
```

Comment:
This does nothing right now, so we probably want to comment it out but it's in the designs.